os/Kconfig

```diff
@@ -315,7 +315,8 @@ config BINARY_SIGNING
 
 config USER_SIGN_PREPEND_SIZE
 	int "Prepended user binary signing size"
-	default 32
+	default 32 if !ARCH_CHIP_AMEBASMART 
+	default 4096 if ARCH_CHIP_AMEBASMART
```

Comment:
Is this configurable for user? If this is fixed for each board, how about hiding this and move this into board or chip Kconfig?